src/api/core/tokenManager.ts

```diff
@@ -68,6 +71,14 @@ export const AuthOptions = new InjectionToken<IAuthOptions>("IAuthOptions");
  * Internal interface of the authorization token
  */
 export interface IAuthToken {
+  /**
+   * Alias for the token pair
+   */
+  auth: string;
+  /**
+   * Is it default token pair?
+   */
+  default: boolean;
```

Comment:
I'd rather name it as `isDefault`

Reply:
ha-ha, I suddenly realised that I don't need this interface anymore 👍

Reply:
that's even better :smile: